src/utils.js

```diff
@@ -0,0 +1,9 @@
+export const randomKeyGenerator = () => {
```

Comment:
We can be more specific with the naming for more clarity. Maybe something like`generateRoomCode`?